docs/CustomUIA.md

```diff
@@ -0,0 +1,115 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT Licence. -->
+
+## Custom UI Automation properties
+Axe Windows supports the inspection of [custom UI Automation properties](https://docs.microsoft.com/en-gb/windows/win32/winauto/uiauto-custompropertieseventscontrolpatterns). The custom properties for which Axe Windows should register are described in a user-supplied text file that is applied to all inspected UI Automation elements.
+
+### Configuration file format
+Custom UI Automation configuration data is specified in a [JSON](https://en.wikipedia.org/wiki/JSON#Syntax) formatted text file. The file consists of one object with the following possible attributes:
```

Comment:
For JSON files intended for direct human consumption, I'd recommend including an allowance for comments in the file as part of our spec. `System.Text.Json` and `Newtonsoft.Json` both support reading files with comments so this shouldn't cause any extra implementation burden.

Reply:
(GitHub and VS Code both understand "JSON with Comments" as a separate language using extension "jsonc", if you want to include comments in any of the examples as a demonstration)

Reply:
Thanks, @dbjorge! I didn't know about NewtonSoft supporting comments. Based on my experiments, it looks like we can use either `//` for single-line comments or `/*...*/` for multi-line comments

Reply:
If we don't need to be strictly json compliant, how about [JSON5](https://json5.org/)?

Reply:
json5 may not be completely supported by Newtonsoft or GitHub, so let's not commit to json5 at this time